controller/linodemachine_controller_helpers_test.go

```diff
@@ -160,39 +168,6 @@ func TestSetUserData(t *testing.T) {
 				}}, nil)
 			},
 		},
-		{
```

Comment:
fwiw I think this is still a valid test case, I hope we don't hit it often but userdata can be too large still

Reply:
fair, just need to figure out how to make data that gzips to more than 16kB

Reply:
maybe we could decrease the limit for the sake of the test to validate a smaller gzip gets rejected? not sure if that would require too much refactoring to handle